.github/actions/validate_api.yaml

```diff
@@ -0,0 +1,19 @@
+name: Validate API
+
+on:
+  pull_request:
+    branches: [ main, linting]
```

Comment:
@wtrocki we don't have the `linting` branch

Reply:
That was used to test it on my fork. Removing